sky/clouds/service_catalog/data_fetchers/fetch_aws.py

```diff
@@ -292,9 +313,14 @@ def fetch_availability_zone_mappings() -> pd.DataFrame:
         action='store_true',
         help='Fetch all global regions, not just the U.S. ones.')
     parser.add_argument(
-        '--no-az-mappings',
+        '--az-mappings',
+        dest='az_mappings',
         action='store_true',
-        help='Not fetching the mapping of availability zone ID to name.')
+        help='Fetch the mapping from availability zone IDs to zone names.')
+    parser.add_argument('--no-az-mappings',
+                        dest='az_mappings',
+                        action='store_false')
```

Comment:
Q/nit: does argparse not automatically add this? absl does for every bool arg.

Reply:
No, unfortunately, `argparse` requires manually adding the negative option. https://stackoverflow.com/questions/15008758/parsing-boolean-values-with-argparse